polkadot/node/core/approval-voting/src/lib.rs

```diff
@@ -119,6 +119,10 @@ const APPROVAL_CACHE_SIZE: u32 = 1024;
 
 const TICK_TOO_FAR_IN_FUTURE: Tick = 20; // 10 seconds.
 const APPROVAL_DELAY: Tick = 2;
+
+/// Interval to check if no show was covered by approval
+/// of the same candidate on a different fork.
+const CHECK_NO_SHOW_COVERED: Tick = 36;
```

Comment:
Shouldn't we compute this value based on config `noShowSlots` ?

Reply:
It doesn't really matter, if we end up on this case the no-show already happened, this is acting as a guarantee that if  we never receive any other assignment and approval in the context of this block_hash, we are just going to check it every now and then in case it got approved in the context of a different fork.

So this needs to be just a value large enough, we don't unnecessarily trigger it.

Reply:
I guess the question is what happens if we adjust the no_show_slots in the runtime to be larger than this value in the future?

Reply:
Nothing happens, because this fix applies only if don't have any-no show that you need to take into account.

Reply:
outdated.